maquetado/setup/scss/tech.scss

```diff
@@ -0,0 +1,51 @@
+$tech-logo-size: 100px;
+$tech-container-margin: 50px;
+$tech-container-height: 300px;
+$tech-container-width: 210px;
+
+h2 {
+  font-size: $text-medium;
+}
+
+.tech {
+  align-items: center;
+  background-color: $green;
+  border: 7px;
+  border-radius: 5px;
+  display: flex;
+  flex-flow: column wrap;
```

Comment:
I dont think you will use elements that need to be wrapped, I would go just with `flex-direction`